src/components/action/features/FeatureAction.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import Action from '../Action';
+import FeaturesSettings from './FeaturesSetings';
+
+class FeaturesAction extends Component {
+    render() {
+        console.log(this.props);
+        return (
+            <Action
+                title="Extract Features"
+                collapsible
+                checked={this.props.checked}
+                onToggleActive={this.props.onToggleActive}
+            >
+                <FeaturesSettings {...this.props} />
+            </Action>
+        );
+    }
+}
+
```

Comment:
Don't forget about propTypes, they are 'signature' of your component. Normally you start writing a component with defining propTypes (of course you maintain them while you're re-designing component, but the overhead is not that critical)